view/frontend/web/template/payment/omise-cc-form.html

```diff
@@ -48,7 +48,7 @@ <h4><!-- ko i18n: 'Select a card to proceed' --><!-- /ko --></h4>
                     <!-- ko foreach: getCustomerCards() -->
                         <li style="list-style: none; margin-bottom: 2rem;">
                             <input name="payment[omise_card]" type="radio"
-                                data-bind="attr: {id: 'Card' + value}, checked: $parent.omiseCard, value: value"/>
+                                data-bind="attr: {id: 'Card' + value}, checked: $parent.omiseCard, event: {change : $parent.chargeWithSavedCard}, value: value"/>
```

Comment:
for this expected behaviour:
```
- Click on "Use a new card", This should show input form for credit card
- Click on saved card, this should hide input form for credit card
```
Just wonder If user click `"Use a new card"` after clicked on saved card,  will it toggle between `display: none` and `display: block`? .